.github/workflows/integration.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
         go-version: [1.16.2]
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+    env:
+      GO111MODULE: on
```

Comment:
I'm curious about this. We request Go 1.16 in line 17, and `GO111MODULE` is `on` by default in 1.16. What am I missing?

Reply:
I should remove it, please hold on :) I will finish it and let you guys know.